docs/api/alerts/health.asciidoc

```diff
@@ -0,0 +1,85 @@
+[[alerts-api-health]]
+=== Get alert framework health API
+++++
+<titleabbrev>Get alert framework health</titleabbrev>
+++++
+
+Retrieve the alert framework health status.
```

Comment:
I think we usually use the term "Alerting framework", rather than "alert framework".
Just making sure we're consistent where we can be :)